ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/organization.json

```diff
@@ -5,27 +5,27 @@
   "fields": [
     {
       "field_name": "title_translated",
-      "label": "Title",
+      "label": "Organization name",
+      "extra_label": "Organization name",
+      "input_title": ["Name in Finnish", "Name in Swedish", "Name in English"],
       "preset": "fluent_core_title_translated",
       "only_default_lang_required": true,
-      "form_placeholder": "My Organization",
+      "form_placeholder": "Organization name",
       "form_languages": ["fi", "sv", "en"],
+      "show_url": true,
       "display_snippet": null
     },
-    {
-      "field_name": "name",
-      "label": "URL",
-      "validators": "not_empty unicode name_validator group_name_validator",
-      "form_snippet": "slug.html",
-      "form_placeholder": "my-theme"
-    },
     {
       "field_name": "description_translated",
-      "label": "Description",
-      "preset": "fluent_core_markdown_translated",
+      "label": "Organisation description",
+      "extra_label": "A general, concise, and easy-to-understand description of the organization",
+      "input_title": ["Description in Finnish", "Description in Swedish", "Description in English"],
       "validators": "fluent_text mark_as_modified_in_catalog_if_changed",
```

Comment:
Only default language should be required, now all the languages are required.

Reply:
According to the design in Abstract, the description fields are all marked as required for all languages.

Reply:
You are actually correct, I accidentally looked at the wrong field. :D